website/docs: added content to Welcome page (#4996)
goauthentik/authentik: website/docs/index.md

## What is authentik?

authentik is an open-source Identity Provider, focused on flexibility and versatility. With authentik, site administrators, application developers, and security engineers a dependable and secure solution for authentication in almost any type of environment. There are robust recovery actions available for the users and applications, including user profile and password management. You can quickly edit, deactivate, or even impersonate a user profile, and set a new passowrd for new users or reset an existing password.

You can use authentik in an existing environment to add support for new protocols, so introducing authentik to your current tech stack doesn't present re-architecting challenges. We already support all of the major providers, such as OAuth2, SAML, LDAP, and SCIM, so you can pick the protocol that you need for each application.

The authentik product provides the following consoles:

> * Admin Interface: a visual tool for the creation and management of users and groups, tokens and credentials, application integrations, events, and the Flows that define standard and customizable login and authentication processes. Easy-to-read visual dashboards display system status, recent logins and authentication events, and application usage.

Author: ```suggestion
* **Admin interface**: a visual tool for the creation and management of users and groups, tokens and credentials, application integrations, events, and the Flows that define standard and customizable login and authentication processes. Easy-to-read visual dashboards display system status, recent logins and authentication events, and application usage.
```

> * User Interface: this console view in authentik displays all of the applications and integrations in which you have implemented authentik. Click on the app that you want to work on, and drill down into the specifics for that app.

Reviewer: ```suggestion
* User Interface: this console view in authentik displays all of the applications and integrations in which you have implemented authentik. Click on the app that you want to access to open it, or drill down to edit its configuration in the admin interface
```

> * User Interface: this console view in authentik displays all of the applications and integrations in which you have implemented authentik. Click on the app that you want to access to open it, or drill down to edit its configuration in the admin interface

Author: ```suggestion
* **User interface**: this console view in authentik displays all of the applications and integrations in which you have implemented authentik. Click on the app that you want to access to open it, or drill down to edit its configuration in the admin interface
```

> * Flow panels: [_Flows_](./flow) are the steps by which the various _Stages_ of a login and authetnication process occurs.  A stage represents a single verification or logic step in the sign-on process. authentik allows for the customization and exact definition of these flows.

Reviewer: ```suggestion
* Flows: [_Flows_](./flow) are the steps by which the various _Stages_ of a login and authetnication process occurs.  A stage represents a single verification or logic step in the sign-on process. authentik allows for the customization and exact definition of these flows.
```

> * Flow panels: [_Flows_](./flow) are the steps by which the various _Stages_ of a login and authentication process occurs.  A stage represents a single verification or logic step in the sign-on process. authentik allows for the customization and exact definition of these flows.

Author: ```suggestion
* Flows: [_Flows_](./flow) are the steps by which the various _Stages_ of a login and authentication process occurs.  A stage represents a single verification or logic step in the sign-on process. authentik allows for the customization and exact definition of these flows.
```

> * Flows: [_Flows_](./flow) are the steps by which the various _Stages_ of a login and authentication process occurs.  A stage represents a single verification or logic step in the sign-on process. authentik allows for the customization and exact definition of these flows.

Author: ```suggestion
* **Flows**: [_Flows_](./flow) are the steps by which the various _Stages_ of a login and authentication process occurs.  A stage represents a single verification or logic step in the sign-on process. authentik allows for the customization and exact definition of these flows.
```

## Installation

Refer to the installatin steps in either [Docker-compose](installation/docker-compose) or [Kubernetes](installation/kubernetes).

For more information about configuration, Beta versions, and additional installation options, see our main [Installation](installation) section.

## Screenshots

> authentik can use Light or Dark mode for the Admin interface and the User interface.

Reviewer: The light/dark theme is supported in all 3 interfaces, even though there's only screenshots for two of them

Author: Yeah, I was wondering! I have a .png of the Welcome flow, and would like to addd it, but wanted to get your thought on that first?

Author: Also I can do it in a subsequent PR, no rush now. :-)

Reviewer: Yeah sure, feel free to add that here (make sure your browser is the same size so it matches the dimensions of the other screenshots)

Author: Precisely because I am rather dreadful with graphics and sizings and resolutions, I'll do it in separate PR.

Reviewer: Actually this reminds me of some other change I wanted to do for this, I'll PR those images too

Reviewer: https://github.com/goauthentik/authentik/pull/4997

Author: ohhhh nice images. But... why not add to the PR, to avoid a merge conflict?
